roles/common/tasks/main.yml

```diff
@@ -10,7 +10,7 @@
     - dependencies
 
 - name: Install necessities and nice-to-haves
-  apt: pkg={{ item }} state=installed
+  apt: pkg={{ item }} state=present
```

Comment:
Is it pkg or name?

Reply:
I prefer `name`, but pkg and package are valid aliases. pkg seemed consistent throught the playbook, so I left it as is.

Reply:
Ok, i added a comment to the inconsistent one ☝🏽